src/templates/antora/template.ts

```diff
@@ -286,22 +178,23 @@ const template = (root: Api, toPath: string): void => {
     }
   });
 
-  const nav = getNavFile(sortedTypes);
-  const adocNav = navToAdoc(nav);
+  const indexPage = Util.getNavFile(sortedTypes);
 
-  addPage({
-    filename: '_data/nav.yml',
-    content: YAML.dump(nav)
-  });
+  const navPages = Util.generateNavPages(indexPage, structure);
 
-  addPage({
-    filename: '_data/moxiedoc_nav.adoc',
-    content: adocNav
+  if (structure === 'legacy') {
+    Util.generateIndexPages(indexPage, sortedTypes, memberTemplate, structure)
+      .forEach((pageOutput) => navPages.push(pageOutput));
+  }
+
+  navPages.forEach((page) => {
+    addPage(page);
   });
```

Comment:
```suggestion
  navPages.forEach(addPage);
```
This can be simplified/reduced to just this.

Reply:
Done